lookupapi/ibis.py

```diff
@@ -0,0 +1,106 @@
+"""
```

Comment:
should we push all these methods to ucamlookup?

Reply:
Could do. They do require that ucamlookup inherit the settings defaults system in this app though so it may be best left as a separate story?